tests/integration/test_rest_handler_handler.py

```diff
@@ -85,3 +87,86 @@ def test_403_api_call():
     )
     assert expected_msg.replace("\n", "") in response.text
     assert response.status_code == 500
+
+
+@pytest.mark.parametrize(
+    "value",
+    [
+        "test[name",
+        "test*name",
+        "test\\name",
+        "test]name",
+        "test(name",
+        "test)name",
+        "test?name",
+        "test:name",
+        "toolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnamet"
+        "oolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnameto"
+        "olongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoo"
+        "longnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametool"
+        "ongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolo"
+        "ngnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolon"
+        "gnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolong"
+        "nametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongn"
+        "ametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongnametoolongna"
+        "metoolongnametoolongnamet",
+    ],
+)
+def test_basic_name_validation(value):
+    prohibited_chars = ["*", "\\", "[", "]", "(", ")", "?", ":"]
+    prohibited_names = ["default", ".", ".."]
```

Comment:
We could potentially lose coverage if we add a new prohibited character in the code but not here.

As a solution you might move this list of characters to a const and import it here to ensure consistency.